dags/js/aws/s3-utils.js

```diff
@@ -4,21 +4,15 @@ const io = require('../IO/file-utils');
 const uuidv4 = require('uuid/v4');
 
 
-function createBucket(s3Params) {
+async function createBucket(s3Params) {
   let s3 = this.getS3Client();
-  s3.createBucket(s3Params, (error) => {
-    if (error) throw error;
-  });
+  await s3.createBucket(s3Params).promise();
 }
 
 async function getObject(s3Params) {
   let s3 = this.getS3Client();
   console.log('Getting the following from S3: ', s3Params);
-  let response = await s3.getObject(s3Params, (error) => {
-    if (error) {
-      throw error;
-    }
-  }).promise();
+  let response = await s3.getObject(s3Params).promise();
```

Comment:
Is it necessary to wait for the object before returning? Might be more efficient to return the promise and refactor the caller accordingly.

Reply:
Could try fancy `finally` or `then()` solutions to deal with the `console.log`, but leaving this for now.